src/Foreign/JavaScript/Server.hs

```diff
@@ -40,21 +40,33 @@ import Foreign.JavaScript.Types
 httpComm :: Config -> EventLoop -> IO ()
 httpComm Config{..} worker = do
     env <- getEnvironment
-    let portEnv = Safe.readMay =<< Prelude.lookup "PORT" env
-    let addrEnv = fmap BS.pack $ Prelude.lookup "ADDR" env
-    
-    let config = Snap.setPort      (maybe defaultPort id (jsPort `mplus` portEnv))
-               $ Snap.setBind      (maybe defaultAddr id (jsAddr `mplus` addrEnv))
-               $ Snap.setErrorLog  (Snap.ConfigIoLog jsLog)
-               $ Snap.setAccessLog (Snap.ConfigIoLog jsLog)
-               $ Snap.defaultConfig
+
+    let config = Snap.setErrorLog     (Snap.ConfigIoLog jsLog)
+               $ Snap.setAccessLog    (Snap.ConfigIoLog jsLog)
+               $ forceSSL env Snap.defaultConfig
```

Comment:
```suggestion
               $ maybe (configureHTTP env) configureSSL jsUseSSL
               $ Snap.defaultConfig
```

Could you split the configuration of HTTP and SSL into different functions, `configureHTTP` and `configuresSSL`? This makes the code more clear.